### ISSUE TITLE
Projects with employees are ranked highly

### ISSUE THREAD

**Issue by jasonLaster** [Nagasaki45/krihelinator#100]
I don't know if you can discount this, but companies that work in the open like Facebook, Mozilla or Microsoft with VS Code get a huge bump that is not taken into account.

**Comment by Nagasaki45** [comment]
Do you have an idea how to do this? I can't think of anything except of manually managing a "blacklist".

**Comment by jasonLaster** [comment]
Theres a github org feature that helps here. Eg vscode all of the employees
are in the orgs team

**Comment by Nagasaki45** [comment]
That's true, but so are [jupyter](https://github.com/jupyter) and [nodejs](https://github.com/nodejs). AFAIK their products are not commercial. I'm sure that there are many more examples. Do you think that their rank should be reduced?

I'm trying to look at the list on the Krihelinator homepage and pick projects that enjoy from this bias, but it's not an easy task. For example, what would you say about [kubernetes/kubernetes](http://github.com/kubernetes/kubernetes), [ansible/ansible](http://github.com/ansible/ansible), or [raspberrypi/linux](http://github.com/raspberrypi/linux). In general, the issue here is beyond implementation.
